android-components/components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt

```diff
@@ -114,15 +115,23 @@ class CustomTabsToolbarFeature(
     internal fun init(config: CustomTabConfig) {
         // Don't allow clickable toolbar so a custom tab can't switch to edit mode.
         toolbar.display.onUrlClicked = { false }
-        val readableColor =
+
+        val readableColor = if (updateToolbarBackground) {
             config.toolbarColor?.let { getReadableTextColor(it) } ?: toolbar.display.colors.menu
+        } else {
+            // It's private mode, the readable color needs match the app.
+            // Note: The main app is configuring the private theme, Custom Tabs is adding the
+            // additional theming for the dynamic UI elements e.g. action & share buttons.
+            getColor(context, R.color.fx_mobile_private_text_color_primary)
```

Comment:
so rather than pasting over `fx_mobile_private_text_color_primary`, let's instead use an attribute already available to us! 

`textColorPrimary` is an Android XML style attribute that we then override inside of our [styles file in Fenix](https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/res/values/styles.xml#221), and this is a pattern used elsewhere in AC.

[Example of usage](https://searchfox.org/mozilla-mobile/source/firefox-android/android-components/components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/Extensions.kt#135)

Reply:
Awesome, great suggestion, thanks!